### DISCUSSION TITLE
[FIRMWARE REQUEST] DancePad layout request

### DISCUSSION THREAD

**Discussion by tomo39** [OpenStickCommunity/GP2040-CE#326] [Q&A] [answered]
Thank you for supporting DancePad in "Display Configuration".
When I select "Custom" in the "ButtonLayout" field
Is it possible to change the "Layout Right" (DancePadB) to "Layout Left" (DancePadA)?
I would like to display both in the "Layout Left" layout.

**Comment by TheTrainGoes** [comment]
Hi @tomo39 ,

With our current firmware you would need to compile a custom layout if the pre-done ones are not working for you.  

We have some updates coming to display options which will allow you more flexibility to setup things as you like.

**Reply by tomo39** [reply]
thank you. I couldn't change the layout because I couldn't build an environment to compile it, so I'm looking forward to an update.

**Comment by mikepparks** [comment]
I assume that the ask here is to take the segmented DancePad layouts and merge them into a single layout. What are the expectations with the other layout slot? Will that be set to Blank or do you expect to have other elements in place?

**Reply by tomo39** [reply]
I was playing Double Play in Dance Dance Revolution, so I wanted the display to be two crosses arranged side by side.

**Reply by mikepparks** [reply, marked as answer]
![image](https://github.com/OpenStickCommunity/GP2040-CE/assets/1214033/693279b9-502f-4eb1-9c12-6a8d08680ede)

With our Display update #860 , we will have DDR Solo, Doubles, and PIU layouts included with a few other rhythm game layouts.

**Reply by tomo39** [reply]
Thank you for your reply and photos. It can also do DDR DP well. It's amazing! I made a single version because I thought I couldn't do it until now. I'll try the update soon


![Uploading 20240206_192031.jpg…]()

**Comment by TheTrainGoes** [comment]
This should be addressed / added as of v0.7.8

**Comment by tomo39** [comment]
DDR double display is now possible! !
thank you.
Up until now, the version with a display was for single-player use.

https://www.youtube.com/watch?v=r3vZFN8rPko&t=148s

Now you can create a double controller

**Reply by mikepparks** [reply]
I wanted to make sure I covered all modes of DDR, including Solo. Glad I could help make that work. ご提案ありがとうございます!
